.github/workflows/reuse-linea-besu-package-run-e2e-tests.yml

```diff
@@ -78,9 +78,11 @@ jobs:
       - name: Replace expected traces api version in coordinator config file
         shell: bash
         run: |
+          sed -i 's/^\(expected-traces-api-version-v2=\).*/\1"${{ env.EXPECTED_TRACES_API_VERSION }}"/' config/coordinator/coordinator-docker.config.toml
```

Comment:
Need to add this, as `expected-traces-api-version-v2` was removed from `coordinator-docker-traces-v2-override.config.toml` in this PR